### ISSUE TITLE
Carousel | Tab key malfunction

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#15111]
- Focus to the carousel
- Press tab to navigate between buttons
- Carousel will start to navigate the carousel items, buttons cannot steal focus.

**Comment by Zubair-Ofcl** [comment]
This is the problem with scroll in view. Once start using Tab key, it will navigate to every next actionable if available, bringing page into view without changing current page. Hence once reached on last page using Tab key, navigation button doesn't work as page is already in view.

**Comment by cetincakiroglu** [comment]
Hi,

In the demo, it fails to steal focus because buttons inside the demo content are focusable by default. Setting tabindex to the buttons will remove them from the tabbing order. Unfortunately we cannot interfere with the tabindex values of projected content inside the component, some users might need to focus inside the content. Simply, adding tabindex to the buttons solves the issue. Please see the example: https://stackblitz.com/edit/er3agn?file=src%2Fapp%2Fdemo%2Fcarousel-basic-demo.html

Closing the issue since we cannot reset tabindex values of the projected content. 
Thanks for reporting the issue!
